src/Polymake.jl

```diff
@@ -38,6 +38,13 @@ function Base.showerror(io::IO, ex::PolymakeError)
     print(io, "Exception occured at Polymake side:\n$(ex.msg)")
 end
 
+function check_jlcxx_version(version)
+    current_jlcxx = VersionNumber(unsafe_string(ccall(:cxxwrap_version_string, Cstring, ())))
+    if (version != current_jlcxx)
+        error("""JlCxx version changed, please run `using Pkg; Pkg.build("Polymake");`""")
```

Comment:
What's the advantage of telling the user this over just calling `Pkg.build("Polymake")`?

Reply:
I think it is more friendly to give a reason for such a request. Also it makes it easier to debug problems if anyone complains about such a message.